basic-tomcat/build/dev/params

```diff
@@ -1,2 +1,3 @@
 APPLICATION_NAME=basic-tomcat
 NAMESPACE=basic-tomcat-dev
+IMAGE_STREAM_TAG_NAME=jboss-webserver30-tomcat8-openshift:1.3
```

Comment:
Do we want to go with `jboss-webserver31-tomcat8-openshift:1.0`? Otherwise LGTM

Reply:
@pabrahamsson i'll try it